docs/internal-developers/testing/releases/1070.md

```diff
@@ -0,0 +1,128 @@
+
+5. Click on it and verify that, when opened, it does take the entire height as always.
+
+![imatge](https://github.com/woocommerce/woocommerce-blocks/assets/3616980/ab9b2b6f-07b6-4202-8a6b-a986bee1ddd4)
+
+### Fix SearchListControl styles [#10192](https://github.com/woocommerce/woocommerce-blocks/pull/10192)
+
+1. Create a category with an & in its name. Ie: Clothing & Accessories.
+2. Create a new post or page and add the Products by Category block.
+3. In the placeholder, make sure Clothing & Accessories is displayed, instead of Clothing `&amp;` Accessories.
+4. Also verify that as you select categories, the tags added to the "x categories selected" section have a gray background.
+5. Expand a category with children categories and verify the checkboxes have the same size.
+
+Before | After
+--- | ---
+![imatge](https://github.com/woocommerce/woocommerce-blocks/assets/3616980/62efe190-c3e6-4aa5-8cf6-a6882e0188f9) | ![imatge](https://github.com/woocommerce/woocommerce-blocks/assets/3616980/ca5fb734-9b1d-4f35-bd8b-cc0283da4761)
+
+### Mini Cart: Fix styling of the Start Shopping button in the drawer [#10169](https://github.com/woocommerce/woocommerce-blocks/pull/10169)
+
+1. Activate Storefront.
+2. Add the Mini-Cart block to a page.
+3. Visit the page on the frontend and select the (empty) Mini-Cart to toggle the drawer.
+4. Verify the button is displaying correctly.
+
+| Before | After |
+| ------ | ----- |
+|<img width="883" alt="before-2" src="https://github.com/woocommerce/woocommerce-blocks/assets/905781/9d8ca8a5-1662-4ebd-8233-3a2948e70843">|<img width="898" alt="after-2" src="https://github.com/woocommerce/woocommerce-blocks/assets/905781/f7088abc-b17b-42fa-b329-57cd401fbe79">|
+
+### Mini Cart block > Update block registration to rely on a metadata file. [#10168](https://github.com/woocommerce/woocommerce-blocks/pull/10168)
+
+1. Create a new post
+2. Make sure the mini-cart block is available for insertion
+3. Insert the block, make changes to its settings and save
+4. On the frontend, make sure everything works as expected: you can add/remove products from the cart and the style changes made in the editor are visible on the frontend.
+
+### Enhance: Bring back user to 1st page when filters are changed [#9952](https://github.com/woocommerce/woocommerce-blocks/pull/9952)
+
+1. Create a new post.
+2. Add all filter blocks and a 'Products Beta' block to it.
+3. Save the post and navigate to the frontend.
+4. Navigate to the 2nd page using pagination. 
+5. Modify any filter and confirm that the page number resets to the 1st page upon filter alteration. 
+6. Repeat the above steps and validate that this also holds true for:
+    - 'Product Collection' block (The product collection block is currently in the experimental phase and is not shipped with the plugin/core)
+    - 'Classic WooCommerce Product Grid Block'
```

Comment:
Can we improve the wording here and update the instructions to give more information on how to test with this `Classic WooCommerce Product Grid Block` ?

Reply:
Sure, I just split this in 2 different scenarios, one for the Products (Beta) block and another for the Classic WooCommerce Product Grid block